ark/utils/segmentation_utils.py

```diff
@@ -104,7 +104,9 @@ def transform_expression_matrix(cell_table, transform, transform_kwargs=None):
             xarray of counts per marker normalized by cell size
     """
     valid_transforms = ['size_norm', 'arcsinh']
-    misc_utils.verify_in_list(transform=transform, valid_transforms=valid_transforms)
+
+    if transform not in valid_transforms:
```

Comment:
This looks like a merge conflict was resolved in the wrong direction; we want to keep the red code